### ISSUE TITLE
Windows Support

### ISSUE THREAD

**Issue by rwilliams** [xeger/backticks#11]
PTY is not available on Windows builds of ruby. Is [something like this](https://github.com/ThrowTheSwitch/Ceedling/commit/d60fe4b236a787c2f464b1ac3e061db21c86610b) relevant to adding Windows support given how backticks currently utilizes PTY?

**Comment by xeger** [comment]
Backticks only uses the PTY module when using unbuffered mode. Unbuffered mode is the default, but if you construct your own Runner, then it's posible to ask for buffered operation, which utilizes `IO.pipe` instead of PTY. I _think_ `IO.pipe` is supported under Win32 MRI, but I lack a Windows system to experiment on.

If you're willing to do some testing for me, I'll produce an experimental branch that "supports" Windows but which only performs buffered execution. You'll need to know how to use Bundler to point to the experimental branch, but that's straightforward. Interested?

**Comment by rwilliams** [comment]
I can test it for you.

**Comment by xeger** [comment]
Great. The branch is named `no-pty` and you can use it by putting this in your Gemfile:

```
gem 'backticks',
  git: 'https://github.com/xeger/backticks',
  branch: 'no-pty'
```

**Comment by rwilliams** [comment]
Awesome. I'll give it a spin this weekend.

**Comment by xeger** [comment]
*Bump* Any luck with the no-pty branch in Windows?

**Comment by rwilliams** [comment]
I haven't been able to rip out all the posix stuff in my gem yet to really thrash backticks yet but so far it looks good. Thanks for your quick patch on this.

**Comment by xeger** [comment]
I published backticks v1.0.1 with your fix; thanks for reporting + testing.
